changelog.md

```diff
@@ -4,20 +4,116 @@ All notable end-user facing changes are documented in this file.
-  More information in [#1682](https://github.com/dotnet/orleans/pull/1682) and in [#1682 (comment)](https://github.com/dotnet/orleans/pull/1682#issuecomment-234371701). 
-- Added new Amazon AWS basic Orleans providers [#2006](https://github.com/dotnet/orleans/issues/2006)
-- Moved Orleans Performance Counters into its own Telemetry Consumer. Now you need to explicitly register the `OrleansPerfCounterTelemetryConsumer` either by code or XML. More information in [#2122](https://github.com/dotnet/orleans/pull/2122) and docs will come later
-	- `Microsoft.Orleans.CounterControl` can still be used to install the performance counters or you can use `InstallUtil.exe OrleansTelemetryConsumers.Counters.dll` to install it without depending on `OrleansCounterControl.exe`
+  More information in [#1682](https://github.com/dotnet/orleans/pull/1682) and in [#1682 (comment)](https://github.com/dotnet/orleans/pull/1682#issuecomment-234371701).
+  - Update ZooKeeperNetEx to 3.4.8.5 #1927
+  - Memory Storage eTag enforcement less strict. #1885
+  - Relational Storage: Added SiloName to Membership table #1787
+  - Added option to perform provider commands quietly #1762
+  - Fix for strange bug in BondSerializer #1790
+  - CreateOrleansTables_SqlServer.sql: Removed support for SQL Server 2000 and 2005 #1779
+  - Added support for using Consul 0.6.0 as a Membership Provider #1267
+  - Allow for extraneous keys in OrleanQuery table #1752
+- Streaming
+  - EventHub receiver metrics reporting abstraction #2146
+  - EventHub stream provider time based message purge #2093
+  - Upgrade to ServiceBus 3.3.2 and related changes. #2087
+  - Add Memory Stream Provider #2063
+  - Persistent stream pulling agent now uses exponential backoff #2078
+  - Add dynamic adding / removing stream providers functionality. #1966
+  - EventHub stream provider logging/monitoring #1857
+  - Updating EventHubAdapterFactory for extensibility. #1861
+  - Consistent implicit subscription Id generation. #1828
+  - ServiceBus upgrade to 3.2.2 #1813
+  - Event hub stream provider EventData to cached data mapping #1727
+  - Refactor of EventHubStreamProvider to better support plugable CacheDataAdapter #1714
+- Bug fixes
+  - Fix SerializationManager.GetFallbackSerializer() in .NET Standard #2211
+  - Fix bug in GrainFactoryServiceProvider #2201
+  - CodeGen: fix generated DeepCopy method to call RecordObject earlier #2135
+  - Fix support for serializing SByte[] #2140
+  - Fix synchronization bug in Orleans/Async/BatchWorker #2133
+  - Fix #2119 by allowing full uninitialization in SiloHost #2127
+  - Persistent Stream Provider initialization timeout fix. #2065
+  - Fix null reference in StreamPubSub grain. #2040
+  - Fix for Comment in configuration XML causes exception to be thrown #1994
+  - Fixed null MethodInfo in Interceptors #1938
+  - Object Pools not pooling fix. #1937 
+  - Fixed EventHub stream provider flow control. #1935
```

Comment:
where these bugs ever introduced in an official release? otherwise we should just not mention them